src/main/scala/is/hail/methods/CalculateConcordance.scala

```diff
@@ -40,53 +40,56 @@ class ConcordanceCombiner extends Serializable {
     }
   }
 
+  def nDiscordant: Long = {
+    var n = 0L
+    for (i <- 2 to 4)
```

Comment:
this is only called once per variant and per sample, so I don't think it's really worth the additional complexity